nunchaku/csrc/flux.h

```diff
@@ -133,6 +133,22 @@ class QuantizedFluxModel : public ModuleWrapper<FluxModel> { // : public torch::
         return hidden_states;
     }
 
+    // expose the norm1 forward method of the transformer blocks
+    // this is used by TeaCache to get the norm1 output
+    std::tuple<torch::Tensor, torch::Tensor, torch::Tensor, torch::Tensor, torch::Tensor> norm1_forward(
```

Comment:
How about renaming this function to norm_one_forward instead of norm1_forward, even though the actual subprocedure called is norm1.forward, as function names would better contain only alphabets?

Reply:
Sure, no problem

Reply:
Renamed